DataFormats/CaloTowers/interface/CaloTower.h

```diff
@@ -198,6 +207,12 @@ class CaloTower : public reco::LeafCandidate {
    GlobalPoint emPosition_;
    GlobalPoint hadPosition_;
 
+  //hcal limits in CT continuous ieta (from CaloTowerTopology)
+  int firstHBRing_, lastHBRing_;
```

Comment:
HcalSubdetector is in the DataFormats subsystem.
So, it's ok to use it, especially if it fits the needs. (I didn't inspect the code enough to understand if that is enough or a tower can be in multiple subdets)